src/styles/blocks/card.scss

```diff
@@ -0,0 +1,77 @@
+.card{
+  box-sizing: border-box;
```

Comment:
Move it to the `*` selector
```suggestion
```

Reply:
> Move it to the `*` selector

Not fixed
UPD. I see you added to the `*` selector, so remove it here as you don't need it anymore